diff --git a/Makefile b/Makefile
index f7cc7d7..70832be 100644
--- a/Makefile
+++ b/Makefile
@@ -139,7 +142,7 @@ $(SHARED_OUTDIR)/$(SHARED_LIB3): $(SHARED_LIBOBJECTS)
 
 endif  # PLATFORM_SHARED_EXT
 
-all: $(SHARED_LIBS) $(SHARED_PROGRAMS) $(STATIC_OUTDIR)/libleveldb.a $(STATIC_OUTDIR)/libmemenv.a $(STATIC_PROGRAMS)
+all: $(SHARED_LIBS) $(STATIC_OUTDIR)/libleveldb.a $(STATIC_OUTDIR)/libmemenv.a
 
 check: $(STATIC_PROGRAMS)
 	for t in $(notdir $(TESTS)); do echo "***** Running $$t"; $(STATIC_OUTDIR)/$$t || exit 1; done
diff --git a/port/port_win.cc b/port/port_win.cc
index 1b0f060..b1800ac 100644
--- a/port/port_win.cc
+++ b/port/port_win.cc
@@ -143,5 +143,7 @@ void AtomicPointer::NoBarrier_Store(void* v) {
   rep_ = v;
 }
 
+uint32_t AcceleratedCRC32C(uint32_t crc, const char* buf, size_t size) { return 0; }
+
 }
 }
