diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -243,8 +243,8 @@
   AC_DEFINE([USE_ANDROID], [1], [Compile with Android specific features enabled])
 
   dnl Check if the Android log library is available.
-  AC_CHECK_HEADERS([android/log.h])
-  AC_SEARCH_LIBS(__android_log_write, [log])
+  #AC_CHECK_HEADERS([android/log.h])
+  #AC_SEARCH_LIBS(__android_log_write, [log])
 
 fi
 
