diff --git a/tor/configure.ac b/tor/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -791,12 +788,7 @@
 struct event_base;
 struct event_base *event_base_new(void);
 void event_base_free(struct event_base *);],
-    [
-#ifdef _WIN32
-{WSADATA d; WSAStartup(0x101,&d); }
-#endif
-event_base_free(event_base_new());
-], [--with-libevent-dir], [/opt/libevent])
+    [;], [--with-libevent-dir], [/opt/libevent])

 dnl Determine the incantation needed to link libevent.
 save_LIBS="$LIBS"
@@ -921,7 +913,7 @@
     dnl We look for SSL_cipher_get_id() because it is present in
     dnl OpenSSL >=1.0.1, because it is not deprecated, and because Tor
     dnl depends on it.
-    [if (getenv("THIS_SHOULDNT_BE_SET_X201803")) SSL_CIPHER_get_id((void *)0);], [],
+    [;], [--with-openssl-dir],
     [/usr/local/opt/openssl /usr/local/openssl /usr/lib/openssl /usr/local/ssl /usr/lib/ssl /usr/local /opt/openssl])

 dnl XXXX check for OPENSSL_VERSION_NUMBER == SSLeay()
@@ -1055,7 +1047,7 @@
 TOR_SEARCH_LIBRARY(zlib, $tryzlibdir, [-lz],
     [#include <zlib.h>],
     [const char * zlibVersion(void);],
-    [zlibVersion();], [--with-zlib-dir],
+    [;], [--with-zlib-dir],
     [/opt/zlib])

 if test "$enable_static_zlib" = "yes"; then
